Assets/GraphSystem.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TorbuTils.Giraphe;
+
+namespace Immerse.BfhClient.Game
+{
+    public class GraphSystem : MonoBehaviour
+    {
+        public static GraphSystem Instance { get; private set; }
+
+        [SerializeField] private GraphVisualizer neighbourVisualizer;
+        [SerializeField] private GraphVisualizer boatVisualizer;
+        [SerializeField] private GraphVisualizer landVisualizer;
+
+        private Graph neighbourGraph;  // Connects neighbouring regions
+        private Graph boatGraph;  // Connects dockable regions
+        private Graph landGraph;  // Connects land regions
+        private List<Region> regions;
+
+        private void Awake()
+        {
+            Instance = this;
+        }
+
+        private void Start()
+        {
+            // Find all regions and assign IDs.
+            // IDs are necessary for the graph package to work
+            regions = new();
+            foreach (Region region in FindObjectsOfType<Region>())
```

Comment:
Kan hende det er lurere å bruke en allerede cachet liste med alle regionene i stedet for å finne dem på nytt. Men dette funker for nå.

Reply:
Yeah kunn ikke finn no anna sted vi brukt ID for regiona, som æ egentlig vil si e veldig viktig for modifiability, men akkurat no burd herre funk